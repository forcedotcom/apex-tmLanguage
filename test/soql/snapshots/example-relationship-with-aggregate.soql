SELECT Name, (SELECT CreatedBy.Name FROM Notes) FROM Account;

SELECT Amount, Id, Name, (SELECT Quantity, ListPrice, PricebookEntry.UnitPrice, PricebookEntry.Name FROM OpportunityLineItems) FROM Opportunity
