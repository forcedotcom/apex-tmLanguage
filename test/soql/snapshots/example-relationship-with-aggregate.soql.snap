>SELECT Name, (SELECT CreatedBy.Name FROM Notes) FROM Account;
#^^^^^^ source.soql keyword.operator.query.select.apex
#      ^ source.soql
#       ^^^^ source.soql keyword.query.field.apex
#           ^ source.soql punctuation.separator.comma.apex
#            ^ source.soql
#             ^ source.soql punctuation.parenthesis.open.apex
#              ^^^^^^ source.soql keyword.operator.query.select.apex
#                    ^ source.soql
#                     ^^^^^^^^^^^^^^ source.soql keyword.query.field.apex
#                                   ^ source.soql
#                                    ^^^^ source.soql keyword.operator.query.from.apex
#                                        ^ source.soql
#                                         ^^^^^ source.soql storage.type.apex
#                                              ^ source.soql punctuation.parenthesis.close.apex
#                                               ^ source.soql
#                                                ^^^^ source.soql keyword.operator.query.from.apex
#                                                    ^ source.soql
#                                                     ^^^^^^^ source.soql storage.type.apex
#                                                            ^^ source.soql
>
>SELECT Amount, Id, Name, (SELECT Quantity, ListPrice, PricebookEntry.UnitPrice, PricebookEntry.Name FROM OpportunityLineItems) FROM Opportunity
#^^^^^^ source.soql keyword.operator.query.select.apex
#      ^ source.soql
#       ^^^^^^ source.soql keyword.query.field.apex
#             ^ source.soql punctuation.separator.comma.apex
#              ^ source.soql
#               ^^ source.soql keyword.query.field.apex
#                 ^ source.soql punctuation.separator.comma.apex
#                  ^ source.soql
#                   ^^^^ source.soql keyword.query.field.apex
#                       ^ source.soql punctuation.separator.comma.apex
#                        ^ source.soql
#                         ^ source.soql punctuation.parenthesis.open.apex
#                          ^^^^^^ source.soql keyword.operator.query.select.apex
#                                ^ source.soql
#                                 ^^^^^^^^ source.soql keyword.query.field.apex
#                                         ^ source.soql punctuation.separator.comma.apex
#                                          ^ source.soql
#                                           ^^^^^^^^^ source.soql keyword.query.field.apex
#                                                    ^ source.soql punctuation.separator.comma.apex
#                                                     ^ source.soql
#                                                      ^^^^^^^^^^^^^^^^^^^^^^^^ source.soql keyword.query.field.apex
#                                                                              ^ source.soql punctuation.separator.comma.apex
#                                                                               ^ source.soql
#                                                                                ^^^^^^^^^^^^^^^^^^^ source.soql keyword.query.field.apex
#                                                                                                   ^ source.soql
#                                                                                                    ^^^^ source.soql keyword.operator.query.from.apex
#                                                                                                        ^ source.soql
#                                                                                                         ^^^^^^^^^^^^^^^^^^^^ source.soql storage.type.apex
#                                                                                                                             ^ source.soql punctuation.parenthesis.close.apex
#                                                                                                                              ^ source.soql
#                                                                                                                               ^^^^ source.soql keyword.operator.query.from.apex
#                                                                                                                                   ^ source.soql
#                                                                                                                                    ^^^^^^^^^^^ source.soql storage.type.apex
>